.github/workflows/cicd.yml

```diff
@@ -0,0 +1,25 @@
+name: CI/CD
+
+on: [push]
```

Comment:
Do we want to run this on every "push"? i.e. this would run if you push on a branch that isn't in a PR, resulting in unnecessary runs.

Running it on `pull_request` I think would make more sense, as it would run when opening the PR and any time you push new code to the pr (`synchronize` trigger), here's some docs: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request